### ISSUE TITLE
Various errors

### ISSUE THREAD

**Issue by Nastras** [oretman/homebridge-mi-flora-care#2]
Hello @oretman,
i just tested your plugin in the hope that it runs better than the previous flower-care plugins. Unfortunately I get several errors in the journalctl.

![F2285B85-F181-483F-ADC7-ADC40570CEE2](https://user-images.githubusercontent.com/27686185/120117801-44948400-c18f-11eb-94cb-6d97439868fb.jpeg)

Would be nice if you could fix it? If I can help you or you need more information let me know.

Thanks!

**Comment by MattB1981** [comment]
I also get this bug and after a bit it builds up after a few hours post boot up and only returns the error.

**Comment by Hucksleyy91** [comment]
I updated my plant sensor and tried this plugin but get the same error 

it worked before I updated the sensor 

![882A9003-939C-4A41-910A-7E3208C4E85C](https://user-images.githubusercontent.com/38526744/146660978-3159e5c2-d793-4889-a8e7-0533cc3c14df.png)

**Comment by Novi001** [comment]
Hi, everyone,

could someone find a solution here?

BG
Novi

**Comment by Novi001** [comment]
Thx @hakla - the code is perfect. No mistakes

**Comment by Aekung** [comment]
> Thx @hakla - the code is perfect. No mistakes

Can you tell me how did you manage to update to @hakla PR?

Thanks in advance.

**Comment by Novi001** [comment]
> > Thx @hakla - the code is perfect. No mistakes
> 
> Can you tell me how did you manage to update to @hakla PR?
> 
> Thanks in advance.

Hi Aekung,

I searched for the device ID of a sensor on my PI4 and found the right file- the search takes a while. Then I manually adopted @hakla's changes.

I searched with the following command:

cd / && grep -r --exclude=".*" "xx:xx:xx:xx:xx:xx" * 2>/dev/null | cut -d":" -f1

It certainly wasn't the easiest way, but it worked in the end.

I hope this helps.

Best regards,
Novi
